tests/src/Functional/PersonContentRenderTest.php

```diff
@@ -143,13 +144,30 @@ public function testPersonPage(): void {
 
     $document_group_title = $content->filter('h3.fs-4');
     $this->assertEquals('Curriculum Vitae', $document_group_title->text());
-    $files = $content->filter('.bcl-file');
-    $this->assertEquals(
-      'Person document test',
-      $files->filter('p')->text()
-    );
+    $files = $content->filter('.bcl-file-container');
 
+    $document_file = File::create([
+      'uri' => $this->getTestFiles('text')[0]->uri,
+    ]);
+    $document_file->save();
+    $absolute_file_url = $document_file->createFileUrl(FALSE);
+    $expected_document = [
+      'file' => [
+        'title' => 'Person document test',
+        'language' => 'English',
+        'meta' => '(1 KB - TXT)',
+        'icon' => 'file-text-fill',
+        'url' => $absolute_file_url,
+      ],
+      'translations' => NULL,
+      'link_label' => 'Download',
+    ];
     $this->assertCount(3, $files);
+    $assert = new FilePatternAssert();
+    foreach ($files as $file) {
+      $file_html = $file->ownerDocument->saveHTML($file);
```

Comment:
$file should be a Crawler object, if not, use ->eq(), and then you can use $file->html() to get the html

Reply:
done. thanks